src/firefly/js/charts/ui/ChartPanel.jsx

```diff
@@ -5,12 +5,12 @@ import {wrapResizer} from '../../ui/SizeMeConfig.js';
 import {flux} from '../../Firefly.js';
 import {get, isEmpty, isUndefined} from 'lodash';
 import {reduxFlux} from '../../core/ReduxFlux.js';
-
+import {singleTraceUI} from '../ChartUtil.js';
 import * as ChartsCntlr from '../ChartsCntlr.js';
 
 import DELETE from 'html/images/blue_delete_10x10.png';
 
-
+//LZ 3/06/18 IRSA-1522 Add this constant variable to control if all the toolbar icons are shown or not shown
```

Comment:
@tgoldina 
As both you and Loi commented, I removed the comment left there.   Since you said that you would take care of Time Series and Finderchart, I did not do anything in Time Series and Finderchart.   I will work on both applications in my local branch.  Once it is done, I will make a final push.  Thanks!